SparkPost/2web2ui
# AC-1177: Billing page updates

### What Changed
 - Removed indent from "Changes to Features" section
 - Hide Changes to features if no features being removed
 - Changed features from having to click "Got it" to already being checked

### How To Test
  - Go to `/account/billing/plan` and check the Changes to Features has removed indents + prechecked
   - To see all conditions, do Free -> Starter, Starter to Premier, and Premier to starter

### To Do
 - [ ] Potentially remove some of the complexity of FeatureChangeContext

### Notes
 FeatureChangeContext is more complex because it combines quantity changes (where users would have to leave the page) and acknowledgements (where features will be dropped automatically)
